content/en/tracing/connect_logs_and_traces/nodejs.md

```diff
@@ -29,9 +29,12 @@ const tracer = require('dd-trace').init({
 
 This enables automatic trace ID injection for `bunyan`, `paperplane`, `pino`, and `winston`.
 
+If you haven't done so already, we recommend configuring the NodeJS tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the smoothest
```

Comment:
```suggestion
If you haven't done so already, configure the NodeJS tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the best
```